controllers/contacts/index.js

```diff
@@ -1,8 +1,10 @@
-const { createNewContact, deleteContact, fetchContact, fetchContacts, updateContact } = require("./services.js");
+const { createNewContact, deleteContact, fetchContact, fetchContacts, updateContact, updateStatusContact } = require("./services.js");
```

Comment:
Kod kiedy jest dłuższy niż 80 znaków to możesz go skracać. Bo im dłuższy tym gorzej się czyta. 
czyli robić coś takiego:
const { 
import1, 
import2, 
... 
}